features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -506,6 +506,14 @@
         <configfile finalname="${openhab.conf}/services/zwave.cfg" override="false">mvn:${project.groupId}/openhab-addons-external/${project.version}/cfg/zwave</configfile>
     </feature>
 
+    <feature name="openhab-binding-iec6205621meter" description="IEC 62056-21 meter" version="${project.version}">
```

Comment:
No the meter1 is coming from the configuration. 
In the configuration you can add multiple meters.
When implementing the binding into a new OH2 binding I hope to find good way to allow user to configure multiple meters within PaperUI